shared/actions/profile.js

```diff
@@ -404,13 +412,13 @@ function outputInstructionsActionLink (): AsyncAction {
         openURL(`https://coinbase.com/${profile.username}#settings`)
         break
       case 'twitter':
-        openURL(`https://twitter.com/home?status=${profile.proof}`)
+        openURLIfNotNull(profile.proofText, `https://twitter.com/home?status=${profile.proofText || ''}`, 'twitter url')
```

Comment:
:sheep: For discussion: should the responsibility for checking null be in the caller? Can we type check the `openURL` calls to force null checking into the caller?